### PR TITLE
test: add basic test

### LINKED ISSUE #48: Needs CI
This is used in many places and has added features that are not in marked, so I think it's time for a unit test and CI.

### PR DESCRIPTION
**Pull request by hashrock**
Close #48

### CODE REVIEW

File: test/test.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "text/html",
+  );
+  assertEquals(document?.querySelector("h1")?.textContent, "Heading");
+  assertEquals(document?.querySelectorAll("li")?.length, 3);
+});
+
+Deno.test("Math rendering", async () => {
+  const math = await Deno.readTextFile("./test/fixtures/math.md");
+  const html = render(math, { allowMath: true });
+  const document = new DOMParser().parseFromString(
```
Comment (lino-levan) [inline test/test.ts:22]: Should there be `expected` html for math rendering? It seems like this would be important to avoid regressions.

**Review by lino-levan — CHANGES_REQUESTED** [review]

**Comment by lino-levan** [comment]
Also, in general, I think we probably want a better testing story for this. I think using puppeteer or something like that would be important to make sure the CSS renders correctly as well (one of the big issues with math rendering when I was making that PR was that font rendering did not work and I didn't realize).

**Comment by lino-levan** [comment]
I should clarify that I think (other than my comment), this PR LGTM. Just thinking about a future PR for more expansive tests given how often deno-gfm is used.

File: test/test.ts (hunk 1 above)
Comment (hashrock) [reply test/test.ts:22]: Thanks, let's add snapshot

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: denoland/setup-deno@main
+        with:
+          deno-version: "1.31.1"
```
Comment (lino-levan) [inline .github/workflows/ci.yml:12]: Do we want to pin the version here? Not sure what the standard is accross other Deno repos.

Comment (hashrock) [reply .github/workflows/ci.yml:12]: hmm I don't have a preference. but let's use 1.x

Comment (lino-levan) [reply .github/workflows/ci.yml:12]: Sounds good to me.

Comment (lino-levan) [reply .github/workflows/ci.yml:12]: Looks like most of the other repos do `1.x` or canary

**Review by lino-levan — APPROVED** [review]
LGTM

**Comment by hashrock** [comment]
Thank you! Yeah more extensive testing is needed. Let's make another issue to expand testing. For now, I've only built the foundation for this issue.

**Pull request merged**